target/linux/ipq806x/files/arch/arm/boot/dts/qcom-ipq8064-d7800.dts

```diff
@@ -280,6 +280,18 @@
 	};
 };
 
+&art {
+	compatible = "nvmem-cells";
+
+	macaddr_gmac1: macaddr_gmac1@6 {
+		reg = <0x6 0x6>;
+	};
+
+	macaddr_gmac2: macaddr_gmac2@0 {
+		reg = <0x0 0x6>;
+	};
```

Comment:
Hmm, I'd personally prefer ordering based on the offset here.

BTW, this whole "naming after the interface" will fail when the same address location is used several times (for different interfaces).
I'm not sure how to nicely name these cases, though.

Reply:
We can consider to drop the interface naming and call them mac1 mac2.... What do you think?

Reply:
Being generic wouldn't be the worst approach either, yes. But which is first? If you just follow the flash locations, it might conflict with the numbers used on a label etc.
I'm still thinking whether we would find a nice approach to include the offset/location, as that's probably the only information actually "belonging" to the address itself.

Reply:
(i'm preparing a script to try and check if the conversion can be automated...)
anyway with your suggestion, i'm currently using 

macaddr_art_0 for example (macaddr_*partition name*_*reg offset*

Reply:
Something like that, yes. I still consider it a bit ugly and wonder whether we can find a more appealing alternative.
In any way, if there is only one partition containing the addresses, one could omit the partition name, of course (macaddr_0 or macaddr_0x0).

Reply:
Isn't that too generic? I think a bit of redundancy can be acceptable to directly understand what is the phandle about.